mirgecom/simutil.py

```diff
@@ -507,14 +517,18 @@ def compare_files_vtu(
             raise ValueError("Fidelity test failed: Mismatched data array sizes")
 
         # verify individual values w/in given tolerance
-        print(f"Field: {point_data2.GetArrayName(i)}", end=" ")
+        fieldname = point_data1.GetArrayName(i)
+        print(f"Field: {fieldname}", end=" ")
         for j in range(arr1.GetSize()):
             test_err = abs(arr1.GetValue(j) - arr2.GetValue(j))
             if test_err > max_field_errors[i]:
                 max_field_errors[i] = test_err
-        print(f"Max Error: {max_field_errors[i]}")
+        print(f"Max Error: {max_field_errors[i]}", end=" ")
+        print(f"Tolerance: {configurate(fieldname, field_tol, tolerance)}")
+
+    violation = any([max_field_errors[i] > configurate(fieldname, field_tol,
+                                        tolerance) for i in range(nfields)])
```

Comment:
I think `fieldname` should be `point_data1.GetArrayName(i)` in this violation check.